### DISCUSSION TITLE
scrollintoview for particular Nodes

### DISCUSSION THREAD

**Discussion by harshmcg** [xyflow/xyflow#4518] [Q&A] [answered]
We have multiple nodes on frames, and on each node we have dropdown and in dropdown list we have list of all the nodes, when user will select particular node from dropdown it scroll to that node. 
we do not want to use fitview()  property because it fits everthing on a single frame and size of nodes is very less , if have multiple nodes there. @reactflow @bcakmakoglu

**Comment by Santoshsohani** [comment]
When Selected a Node from the dropdown, Selected node should be focused right..??

**Reply by harshmcg** [reply]
Yes, should be focused and scrolled to that node

**Reply by bcakmakoglu** [reply, marked as answer]
You *can* use `fitView({ nodes: [{ id: '1' }], duration: 1000 })` which will zoom into a specific node (or nodes).
